bin/codeclimate-test-reporter

```diff
@@ -2,23 +2,20 @@
 
 require "codeclimate-test-reporter"
 
-COVERAGE_FILE = "coverage/.resultset.json".freeze
+repo_token = ENV["CODECLIMATE_REPO_TOKEN"]
+if repo_token.nil? || repo_token.empty?
+  STDERR.puts "Cannot post results: environment variable CODECLIMATE_REPO_TOKEN must be set."
+  exit
+end
+
+COVERAGE_FILE = ARGV.first || "coverage/.resultset.json"
 
-if (repo_token = ENV["CODECLIMATE_REPO_TOKEN"]) && !repo_token.empty?
-  if File.exist?(COVERAGE_FILE)
-    begin
-      results = JSON.parse(File.read(COVERAGE_FILE))
-    rescue JSON::ParserError => e
-      $stderr.puts "Error encountered while parsing #{COVERAGE_FILE}: #{e}"
-      exit(1)
-    end
+abort "Coverage results not found" unless File.exist?(COVERAGE_FILE)
```

Comment:
alright. I'd argue that abort is similar to puts and shouldn't be wrapped in parens, so i'll remove them in L18